src/test/tests/operators/displace.py

```diff
@@ -29,7 +29,7 @@
 #
 #    Kathleen Biagas, Mon Dec 19 15:45:38 PST 2016
 #    Use FilledBoundary plot for materials instead of Subset.
-#
+# 
```

Comment:
```suggestion
#
```

Reply:
Actually, should just revert this change that adds a space after the '#'.

Reply:
oops, I'm sorry. I originally added the test to displace but moved it to mili. Did not do a good job cleaning up my mess.